src/wazuh_db/wdb_global.c

```diff
@@ -1402,6 +1402,8 @@ wdbc_result wdb_global_sync_agent_groups_get(wdb_t *wdb, wdb_groups_sync_conditi
     char *out_aux = cJSON_PrintUnformatted(*output);
     size_t response_size = strlen(out_aux);
     os_free(out_aux);
+    // Recent agents will be excluded
```

Comment:
I would reword this comment to `Agents registered recently may be excluded depending on the 'agent_registration_delta' value.`

Reply:
Fixed!